### PR TITLE
[FIX] Apply format style not correct on Mobile/Tablet

### PR DESCRIPTION
**Pull request by dab246**
### Issues
- https://github.com/linagora/tmail-flutter/issues/1064
- https://github.com/linagora/tmail-flutter/issues/1065
- https://github.com/linagora/tmail-flutter/issues/1066

### Resolved

- Mobile iOS

https://user-images.githubusercontent.com/80730648/196609436-da29dc50-15e2-4536-9d0e-593537bcbf35.mov

- Mobile Android

https://user-images.githubusercontent.com/80730648/196609570-0be8050d-4238-465a-bf85-50d7dd538566.mov

- iPad

https://user-images.githubusercontent.com/80730648/196609625-f6d429bd-86a0-46ba-b9f7-a15feffd0fca.mov

### CODE REVIEW

File: lib/views/widgets/dialog/dialog_utils.dart [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    );
+  }
+
+  static void showDialogCenter(BuildContext context, Widget child) {
+    showDialog(
+      context: context,
+      builder: (BuildContext context) {
+        return AlertDialog(
+          contentPadding: EdgeInsets.zero,
+          shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(16)),
```
Comment (sherlockvn) [inline lib/views/widgets/dialog/dialog_utils.dart:42]: I think we can take advantage of const keyword in here, but with a little longer code. 
`const RoundedRectangleBorder(borderRadius: BorderRadius.all(Radius.circular(10)))`

File: lib/views/widgets/color_picker_keyboard.dart [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  @override
+  Widget build(BuildContext context) {
+    return LayoutBuilder(builder: (context, constraint) {
+      return Container(
+        decoration: BoxDecoration(
+            borderRadius: BorderRadius.circular(4),
+            color: Colors.white
+        ),
+        child: ClipRRect(
+          borderRadius: BorderRadius.circular(4),
```
Comment (sherlockvn) [inline lib/views/widgets/color_picker_keyboard.dart:23]: same here with BorderRadius.all() method

File: lib/views/widgets/color_picker_keyboard.dart [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    Key? key,
+    this.onSelected,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return LayoutBuilder(builder: (context, constraint) {
+      return Container(
+        decoration: BoxDecoration(
+            borderRadius: BorderRadius.circular(4),
```
Comment (sherlockvn) [inline lib/views/widgets/color_picker_keyboard.dart:19]: same

File: lib/views/widgets/list_header_style.dart [hunk 4]
```diff
@@ -35,30 +34,40 @@ class ListHeaderStyle extends StatelessWidget {
...
-            child: _buildHeaderStyle(headerStyle.styleName,
-                headerStyle.textSize, headerStyle.fontWeight));
+            child: _buildHeaderStyle(
+                headerStyle.styleName,
+                headerStyle.textSize,
+                headerStyle.fontWeight));
       case HeaderStyleType.code:
         return Container(
             decoration: BoxDecoration(
                 borderRadius: BorderRadius.circular(4),
```
Comment (sherlockvn) [inline lib/views/widgets/list_header_style.dart:47]: same

File: lib/views/widgets/color_picker_keyboard.dart [hunk 5]
```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
```
Comment (ManhNTX) [inline lib/views/widgets/color_picker_keyboard.dart:1]: Screen Shot 2022-10-19 at 14 28 48

It seems to be wrong compared to the design. Can you recheck it?

**Comment by hoangdat** [comment]
LGTM

File: lib/views/widgets/color_picker_keyboard.dart (hunk 5 above)
Comment (dab246) [reply lib/views/widgets/color_picker_keyboard.dart:1]: IMO, it's not so important. It's just different layout.

File: lib/views/widgets/dialog/dialog_utils.dart (hunk 1 above)
Comment (dab246) [reply lib/views/widgets/dialog/dialog_utils.dart:42]: Oh, it's the same you. You can use two ways if you want.

File: lib/views/widgets/color_picker_keyboard.dart (hunk 2 above)
Comment (dab246) [reply lib/views/widgets/color_picker_keyboard.dart:23]: same

File: lib/views/widgets/color_picker_keyboard.dart (hunk 3 above)
Comment (dab246) [reply lib/views/widgets/color_picker_keyboard.dart:19]: same

File: lib/views/widgets/list_header_style.dart (hunk 4 above)
Comment (dab246) [reply lib/views/widgets/list_header_style.dart:47]: same

**Review by hoangdat — APPROVED** [review]

File: lib/views/widgets/color_picker_keyboard.dart (hunk 5 above)
Comment (dab246) [reply lib/views/widgets/color_picker_keyboard.dart:1]: I updated it. Please check again. @ManhNTX 

Simulator Screen Shot - iPhone 14 Pro Max - 2022-10-20 at 11 14 52

**Review by ManhNTX — DISMISSED** [review]
LGTM

File: lib/views/widgets/mobile/rich_text_option.dart [hunk 6]
```diff
@@ -380,60 +311,53 @@ class RichTextOption extends StatelessWidget {
...
-                  richTextController.dentTypeApply.value == DentType.outdent,
-                  () {
-                richTextController.selectDentTypeType(
-                    DentType.outdent, context);
-              }, _imagePaths.icOutDentFormat),
-            ],
-          );
-        }));
+      valueListenable: richTextController.dentTypeApply,
+      builder: (context, value, _) {
```
Comment (sherlockvn) [inline lib/views/widgets/mobile/rich_text_option.dart:315]: value can be _ here

File: lib/views/widgets/mobile/rich_text_option.dart [hunk 7]
```diff
@@ -380,60 +311,53 @@ class RichTextOption extends StatelessWidget {
...
-                  richTextController.orderListTypeApply.value ==
-                      OrderListType.numberedList, () {
-                richTextController.selectOrderListType(
-                    OrderListType.numberedList, context);
-              }, _imagePaths.icNumberOrder),
-            ],
-          );
-        }));
+      valueListenable: richTextController.orderListTypeApply,
+      builder: (context, value, _) {
```
Comment (sherlockvn) [inline lib/views/widgets/mobile/rich_text_option.dart:338]: same

File: lib/views/widgets/tablet/option_container_for_tablet.dart [hunk 8]
```diff
@@ -12,7 +12,8 @@ class OptionContainerForTablet extends StatelessWidget {
     required this.title,
     required this.child,
     this.titleBack,
-    this.padding = const EdgeInsets.fromLTRB(16, 16, 16, 32),
+    this.padding = const EdgeInsets.only(left: 24, right: 24, top: 24, bottom: 35),
+    this.maxWidth = 320,
     required this.richTextController,
```
Comment (sherlockvn) [inline lib/views/widgets/tablet/option_container_for_tablet.dart:17]: i think move this line to upper will have better look :)

File: lib/views/widgets/rich_text_keyboard_toolbar.dart [hunk 9]
```diff
@@ -23,27 +26,33 @@ class RichTextKeyboardToolBar extends StatelessWidget {
...
     super.key,
     this.insertImage,
     this.insertAttachment,
     this.isLandScapeMode,
-    required this.backgroundKeyboardToolBarColor,
+    this.backgroundKeyboardToolBarColor = CommonColor.colorBackgroundToolBar,
     this.heightToolBar = defaultKeyboardToolbarHeight,
+    this.paddingToolbar,
+    this.paddingIcon,
     required this.richTextController,
```
Comment (sherlockvn) [inline lib/views/widgets/rich_text_keyboard_toolbar.dart:44]: same

**Review by hoangdat — APPROVED** [review]

**Pull request merged**